go/vt/binlog/binlogplayer/binlog_player_test.go

```diff
@@ -395,7 +396,7 @@ func TestCreateVReplicationKeyRange(t *testing.T) {
 	}
 
 	got := CreateVReplication("Resharding", &bls, "MariaDB/0-1-1083", throttler.MaxRateModuleDisabled, throttler.ReplicationLagModuleDisabled, 481823, "db", 0, 0, false)
-	if got != want {
+	if textutil.Normalize(got) != textutil.Normalize(want) {
```

Comment:
This does SQL comparison, so why does this try to do the normalize thing? This comment applies to many more cases, it seems to be added in places that don't deal with JSON / protobuf?